### PR TITLE
configuration files for Ferrier-Aligo MP scheme

### PR DESCRIPTION
**Pull request by mzhangw**
This PR includes configuration files to run Ferrier-Aligo MP scheme with c768 regional setup. 

The associated PRs are:
https://github.com/NCAR/GFDL_atmos_cubed_sphere/pull/5
https://github.com/NCAR/fv3atm/pull/2
https://github.com/NCAR/ccpp-physics/pull/352 (originally, now replaced by https://github.com/NCAR/ccpp-physics/pull/358)

### CODE REVIEW

**Comment by grantfirl** [comment]
@mzhangw Could you please add links to associated PRs in a comment (for all PRs)? This makes it easier for reviewers to follow changes and it documents the work better. Also, choose one of the PRs to try to describe the work done as completely as you can to supplement reviewers' understanding of the changes (so we don't just rely on reading code). The RT logs should be uploaded to the same PR where you have your more complete description. Thanks!

File: tests/tests/fv3_ccpp_regional_c768_FA_update_moist [hunk 1]
```diff
@@ -0,0 +1,25 @@
+###############################################################################
+#
+#  FV3 CCPP regional c768 test
```
Comment (grantfirl) [inline tests/tests/fv3_ccpp_regional_c768_FA_update_moist]: You may want to update the description to mention that this is testing the Ferrier-Aligo scheme.

File: tests/rt_ccpp_HAFS.conf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Compile with CCPP - dynamic mode
+COMPILE | CCPP=Y REPRO=Y                  | standard    | hera.intel     |             |
+# Run tests, input data currently only available on Theia
+#RUN    | fv3_ccpp_ferhires               | standard    |                |             |
+RUN    | fv3_ccpp_regional_c768_FA_update_moist   | standard    |                |             |
```
Comment (grantfirl) [inline tests/rt_ccpp_HAFS.conf]: So, for the merge, the new fv3_ccpp_regional_c768_FA_update_moist test should be added to a different rt*.conf? The CCPP section of rt.conf?

File: tests/fv3_conf/ccpp_regional_FA_run.IN [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.fv_core.res.tile1.nc    INPUT/fv_core.res.tile1.nc
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.fv_srf_wnd.res.tile1.nc INPUT/fv_srf_wnd.res.tile1.nc
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.fv_tracer.res.tile1.nc  INPUT/fv_tracer.res.tile1.nc
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.phy_data.nc             INPUT/phy_data.nc
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.sfc_data.nc             INPUT/sfc_data.nc
+fi
+cp    @[RTPWD]/FV3_input_data/DETAMPNEW_DATA* .
+cp    @[RTPWD]/FV3_input_data/*grb .
+cp    @[RTPWD]/FV3_input_data/*_table .
+cp    /home/Man.Zhang/diag_table_FA diag_table
```
Comment (grantfirl) [inline tests/fv3_conf/ccpp_regional_FA_run.IN]: This is pointing to Man's personal directory. Should it point somewhere else?

File: parm/ccpp_regional_c768_FA.nml.IN [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+ &gfs_physics_nml
+       fhzero         = 3.
+       ldiag3d        = .false.
+       fhcyc          = 24.
+       nst_anl        = .true.
+       use_ufo        = .true.
+       pre_rad        = .false.
+       ncld           = 5
+       imp_physics    = 15
+       spec_adv       = .true.
```
Comment (grantfirl) [inline parm/ccpp_regional_c768_FA.nml.IN]: Are spec_adv and RHGRD the only new namelist variables for F-A?

File: tests/fv3_conf/ccpp_regional_FA_run.IN (hunk 3 above)
Comment (climbfuji) [reply tests/fv3_conf/ccpp_regional_FA_run.IN]: This should be staged in the gmtb area, independent of a user, until it gets moved into the operational regression test baseline maintained by EMC.

File: tests/rt_ccpp_HAFS.conf (hunk 2 above)
Comment (climbfuji) [reply tests/rt_ccpp_HAFS.conf]: This should be added to rt_ccpp_dtc.conf for the time being, no new rt_ccpp_HAFS.conf please.

**Comment by mzhangw** [comment]
You have suggested using rt_ccpp_dtc.conf . It is OK with me. but no
benchmark is necessary considering EMC developers are still tuning now.




I use these to test:


# Compile with CCPP - dynamic mode

COMPILE | CCPP=Y REPRO=Y                  | standard    | hera.intel     |
            |

# Run tests, input data currently only available on Theia

RUN    | fv3_ccpp_regional_c768_FA_update_moist   | standard    |
      |

**Comment by climbfuji** [comment]
Ok, I will add this in one of the next PRs. Please no more changes except the tiny removal of comments to avoid having to rerun the tests. If addressing Grant's changes requires rerunning the tests, then we don't have any other choice, though.

**Pull request merged**